corehq/apps/couch_sql_migration/management/commands/migrate_domain_from_couch_to_sql.py

```diff
@@ -250,11 +275,11 @@ def _highlight(text):
         sep = "|" if ids_in_couch == ids_in_sql else "≠"
         doc_count_row = row.format(n_couch, sep, n_sql)
 
-        print('\n{:_^79}'.format(" %s " % name))
+        print('\n{:_^79}'.format(f" {name} "))
```

Comment:
Regarding your suggestion, @millerdev, I'd be interested in your opinion about nested f-strings and readability. I chose not to nest them because I found that less readable, but maybe that's just because I haven't been using them for long, and my opinion might change over time.